scripts/start.sh

```diff
@@ -28,12 +28,16 @@ if [ -n "$INGRESS_BASE_DOMAIN" ]; then
   sed -i "s|ingressBaseDomain: .*|ingressBaseDomain: '$INGRESS_BASE_DOMAIN',|" /www/index.html
 fi
 
-if [ -n "$AWS_CAPABILITIES_JSON" ] && [ "$PROVIDER" = "aws" ]; then
-  sed -i "s|awsCapabilitiesJSON: .*|awsCapabilitiesJSON: '$AWS_CAPABILITIES_JSON',|" /www/index.html
+if [ "$PROVIDER" = "aws" ]; then
+  if [ -n "$AWS_CAPABILITIES_JSON" ]; then 
+    sed -i "s|awsCapabilitiesJSON: .*|awsCapabilitiesJSON: '$AWS_CAPABILITIES_JSON',|" /www/index.html
+  fi
 fi
```

Comment:
I can't get how it could be wrong:

```
sh-3.2$ AWS_CAPABILITIES_JSON='a'
sh-3.2$ PROVIDER=aws
sh-3.2$ [ -n "$AWS_CAPABILITIES_JSON" ] && [ "$PROVIDER" = "aws" ] ; echo $?
0
sh-3.2$ PROVIDER=azure
sh-3.2$ [ -n "$AWS_CAPABILITIES_JSON" ] && [ "$PROVIDER" = "aws" ] ; echo $?
1
sh-3.2$ PROVIDER=aws
sh-3.2$ [ -n "$AWS_CAPABILITIES_JSON" ] && [ "$PROVIDER" = "aws" ] ; echo $?
0
sh-3.2$ AWS_CAPABILITIES_JSON=''
sh-3.2$ [ -n "$AWS_CAPABILITIES_JSON" ] && [ "$PROVIDER" = "aws" ] ; echo $?
1
```